solr/core/src/test-files/solr/collection1/conf/schema-spellchecker.xml

```diff
@@ -49,9 +49,22 @@
     </analyzer>
   </fieldType>
 
+  <fieldType name="teststop_type" class="solr.TextField">
```

Comment:
I haven't checked in more detail, but don't we normally camelCase?   testStopType ???

Reply:
I saw that in different test files, the fields and field types appear in either camel case or snake case (an sometimes even in kebab case).

I personally use snake case, and I believe the Ref Guide examples also suggest snake case (e.g., field types `text_general` or `text_ws`).

Reply:
thanks for taking the time to check the Ref Guide!

Reply:
we probably should `snake_case` it everywhere......   in all of our test-files..  to have a single pattern.